uwblueprint/community-fridge-kw
# feat: admin delete range of checkins

## Brief description. What is this change? 

### [Delete a Date Range of Shifts for Admin Fridge Check-ins](https://www.notion.so/uwblueprintexecs/Delete-a-Date-Range-of-Shifts-for-Admin-Fridge-Check-ins-942f095f24c74279bb2bffe86a84add0)



## Implementation description. How did you make this change?
* Added route called `/admin/delete-check-ins`
* Added `DeleteCheckIns.tsx` - similar to [create checkin page](https://github.com/uwblueprint/community-fridge-kw/pull/176)

<img width="787" alt="image" src="https://user-images.githubusercontent.com/69697180/162367629-bd6034f9-dbb2-4e92-b613-f1e69a556dd3.png">


## Steps to test
1. Create checkins
2. Sign in as admin and go to `/admin/delete-check-ins`
3. Enter date range to delete checkins
   * `startdate` == `enddate`
   * no shifts in given range
   * check that deletion is inclusive of `startdate` and `enddate`

## Checklist
